index.html

```diff
@@ -32,11 +32,11 @@ <h1>I’m Alexandra. Glad to see you!</h1>
         contact me.</p>
       <div class="third-text">LET’S CONNECT</div>
       <ul class="media-icon">
-        <li><a href="#"><img src="img/twitter.png"></a></li>
-        <li><a href="#"><img src="img/Linkedin.png"></a></li>
-        <li><a href="#"><img src="img/m.png"></a></li>
-        <li><a href="#"><img src="img/github.png"></a></li>
-        <li><a href="#"><img src="img/fingers.png"></a></li>
+        <li><a href="#"><img src="img/twitter.png" alt="twitter"></a></li>
+        <li><a href="#"><img src="img/Linkedin.png" alt="linkedin"></a></li>
+        <li><a href="#"><img src="img/m.png" alt="media"></a></li>
```

Comment:
- [ ] Kindly make sure you're using the appropriate alternative text, the `alt` text here should be "Medium" and not "media" this could confuse the user. ✔